sdk/core/azure-core/azure/core/serialization.py

```diff
@@ -567,6 +582,37 @@ def _deserialize(cls, data):
             return cls(data)
         return mapped_cls._deserialize(data)  # pylint: disable=protected-access
 
+    def as_dict(self, *, exclude_readonly: bool = False, exclude_none: bool = False) -> typing.Dict[str, typing.Any]:
```

Comment:
`json.dumps` will return a string and need to loads again to get dict. I prefer to use more efficient way.